tests/ops/models/test_privacy_request.py

```diff
@@ -669,3 +671,31 @@ def test_can_run_if_no_saved_checkpoint(self):
             )
             is True
         )
+
+
+def test_consent(db):
+    privided_identity_data = {
+        "privacy_request_id": None,
+        "field_name": "email",
+        "encrypted_value": {"value": "test@email.com"},
+    }
+    provided_identity = ProvidedIdentity.create(db, data=privided_identity_data)
```

Comment:
You could clean up these resources at the end of your test by deleting the `provided_identity` and also verify then that both consent resources are created - a good opportunity to protect your cascade delete functionality with a unit test!

Reply:
In general [this fixture](https://github.com/ethyca/fidesops/blob/5da10cc9f650c6890027be3c6f5c6c2ba6189c26/tests/ops/conftest.py#L107) will do the clean up, but I didn't consider the cascade delete test angle. I'll add it.

Reply:
that's right, I forgot you added this cleanup fixture.  I was thinking we still had issues somewhere with collision issues but maybe those are more related to the cache and not the application db.